qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/fitting_widgets/basic_fitting/basic_fitting_presenter.py

```diff
@@ -233,6 +238,39 @@ def handle_evaluation_type_changed(self) -> None:
         """Handle when the evaluation type is changed."""
         self.model.evaluation_type = self.view.evaluation_type
 
+    def handle_plot_guess_type_changed(self) -> None:
+        """Handle when the evaluation type is changed."""
+        self.model.plot_guess_type = self.view.plot_guess_type
+        if self.model.plot_guess_type == 'x from fit range':
```

Comment:
Can these strings ('x from fit range', etc.) be imported from the basic fitting context?